tween.js

```diff
@@ -148,7 +148,45 @@ pc.extend(pc, function () {
 
             return this;
         },
+        
+        by: function (properties, duration, easing, delay, repeat, yoyo) {
+            this._properties = _parseProperties(properties);
+            this.duration = duration;
+            
+            if ('x' in this._properties) 
```

Comment:
Seems a shame to limit this to only vector and color property values? Couldn't we do it more generically so that it works for any value?